Tasks/DownloadPackageV1/package.ts

```diff
@@ -49,7 +49,7 @@ export abstract class Package {
         this.executeWithRetries = builder.ExecuteWithRetries;
     }
 
-    protected abstract async getDownloadUrls(
+    protected abstract  getDownloadUrls(
```

Comment:
I wish we actually rewrite this function to async/await syntax instead of promise syntax, but not blocking on that if we need to get this fix soon :)